test/metabase/lib/js_test.cljs

```diff
@@ -680,3 +680,119 @@
                         :keyword "too"
                         :value   nil}]
       (is (js= expected (lib.js/display-info->js input))))))
+
+(deftest ^:parallel fresh-legacy-query-caching-test
```

Comment:
This test (and others) is marked parallel while using `with-redefs`. I suppose that could be an issue in cljs. Is that ok in cljs?

Reply:
Ah, good catch. I updated these tests to `^:synchronized`. (Though as you say, it doesn't matter in JS.) I also added `cljs.core/with-redefs` and friends to the linter's list of things disallowed in parallel tests. In `:require` etc. namespace that start with `clojure.*` get rewritten to `cljs.*` but that doesn't happen in the linter.